--- a/includes/class-personal-data.php	2026-07-15 21:03:10.274492055 +0000
+++ b/includes/class-personal-data.php	2026-07-15 21:17:21.522315645 +0000
@@ -159,10 +159,9 @@
 						'label'       => esc_html__( 'Enable integration with the WordPress tools for exporting and erasing personal data.', 'gravityforms' ),
 						'tooltip'     => gform_tooltip( 'personal_data_enable', null, true ),
 						'disabled'    => empty( $identification_field_choices ),
-						'after_input' => ! empty( $identification_field_choices ) ? '' : sprintf(
-							'<div class="notice-error gf-notice alert error">%s</div>',
-							esc_html__( 'You must add an email address field to the form in order to enable this setting.', 'gravityforms' )
-						),
+						'description' => empty( $identification_field_choices )
+							? esc_html__( 'You must add an email address field to the form in order to enable this setting.', 'gravityforms' )
+							: '',
 					),
 					array(
 						'name'       => 'exportingAndErasing[identificationField]',
