--- a/includes/class-personal-data.php	2026-07-15 21:03:10.274492055 +0000
+++ b/includes/class-personal-data.php	2026-07-15 21:24:50.683804297 +0000
@@ -159,10 +159,12 @@
 						'label'       => esc_html__( 'Enable integration with the WordPress tools for exporting and erasing personal data.', 'gravityforms' ),
 						'tooltip'     => gform_tooltip( 'personal_data_enable', null, true ),
 						'disabled'    => empty( $identification_field_choices ),
-						'after_input' => ! empty( $identification_field_choices ) ? '' : sprintf(
-							'<div class="notice-error gf-notice alert error">%s</div>',
-							esc_html__( 'You must add an email address field to the form in order to enable this setting.', 'gravityforms' )
-						),
+						'description' => empty( $identification_field_choices )
+							? sprintf(
+								'<div class="alert error"><p>%s</p></div>',
+								esc_html__( 'You must add an email address field to the form in order to enable this setting.', 'gravityforms' )
+							)
+							: '',
 					),
 					array(
 						'name'       => 'exportingAndErasing[identificationField]',
@@ -589,6 +591,16 @@
 				'fields'         => self::settings_fields( $form_id ),
 				'initial_values' => rgar( $form, 'personalData' ),
 				'save_callback'  => array( 'GF_Personal_Data', 'process_form_settings' ),
+				'before_fields'  => function() use ( $form ) {
+					if ( ! empty( self::get_identification_fields_choices( $form ) ) ) {
+						return;
+					}
+
+					printf(
+						'<div class="alert error"><p>%s</p></div>',
+						esc_html__( 'You must add an email address field to the form in order to enable this setting.', 'gravityforms' )
+					);
+				},
 				'after_fields'   => function() {
 					?>
 					<script>
